lstchain/io/lstcontainers.py

```diff
@@ -35,96 +35,81 @@ class DL1ParametersContainer(Container):
-    n_pixels = Field(-1, 'Number of pixels after cleaning')
-    concentration_cog = Field(np.nan, 'Fraction of intensity in three pixels '
-                                      'closest to the cog')
-    concentration_core = Field(np.nan, 'Fraction of intensity inside hillas '
-                                       'ellipse')
-    concentration_pixel = Field(np.nan, 'Fraction of intensity in brightest '
-                                        'pixel')
-    n_islands = Field(-1, 'Number of Islands')
-    alt_tel = Field(None, 'Telescope altitude pointing',
-                    unit=u.rad)
-    az_tel = Field(None, 'Telescope azimuth pointing',
-                   unit=u.rad)
-
-    sin_az_tel = Field(None, 'Sin of telescope azimuth pointing, used for RF')
-
-    obs_id = Field(-1, 'Observation ID')
-    event_id = Field(-1, 'Event ID')
-    calibration_id = Field(-1, 'ID of the employed calibration event')
-    dragon_time = Field(None, 'Dragon time event trigger')
-    ucts_time = Field(None, 'UCTS time event trigger')
-    tib_time = Field(None, 'TIB time event trigger')
-    ucts_jump = Field(False, 'UCTS info jumped (corrected)')
-
-    mc_energy = Field(None, 'Simulated Energy', unit=u.TeV)
-    log_mc_energy = Field(None, 'log of simulated energy/TeV')
-    mc_alt = Field(None, 'Simulated altitude', unit=u.rad)
-    mc_az = Field(None, 'Simulated azimuth', unit=u.rad)
-    mc_core_x = Field(None, 'Simulated impact point x position', unit=u.m)
-    mc_core_y = Field(None, 'Simulated impact point y position', unit=u.m)
-    mc_h_first_int = Field(None, 'Simulated first interaction height', unit=u.m)
-    mc_type = Field(-9999, "MC shower primary ID 0 (gamma), 1(e-),"
-                           "2(mu-), 100*A+Z for nucleons and nuclei,"
-                           "negative for antimatter.")
-    mc_az_tel = Field(None, 'Telescope MC azimuth pointing', unit=u.rad)
-    mc_alt_tel = Field(None, 'Telescope MC altitude pointing', unit=u.rad)
-    mc_x_max = Field(None, "MC Xmax value", unit=u.g / u.cm ** 2)
-    mc_core_distance = Field(None, "Distance from the impact point to the telescope", unit=u.m)
-
-    hadroness = Field(None, "Hadroness")
-    wl = Field(u.Quantity(np.nan), "width/length")
-
-    tel_id = Field(None, "Telescope Id")
-    tel_pos_x = Field(None, "Telescope x position in the ground")
-    tel_pos_y = Field(None, "Telescope y position in the ground")
-    tel_pos_z = Field(None, "Telescope z position in the ground")
-
-    trigger_type = Field(None, "Trigger type")
-    ucts_trigger_type = Field(None, "UCTS trigger type")
-    trigger_time = Field(None, "Trigger time")
-    event_type = Field(None, "Event type")
+    intensity = Field(np.nan, 'Total intensity (size)', dtype=np.float32)
+    log_intensity = Field(np.nan, 'log of total intensity (size)', dtype=np.float32)
+
+    x = Field(np.nan, 'Centroid x coordinate', dtype=np.float32, unit=u.m)
```

Comment:
Default must be the correct dtype and unit, so `u.Quantity(np.float32(np.nan), u.m)` in this case.

We should probably validate this in ctapipe...

Reply:
I would have thought that the `Field` would set the type and unit to the default as well. Making changes then.